packages/kit/types/index.d.ts

```diff
@@ -427,10 +427,12 @@ export interface KitConfig {
 	 */
 	output?: {
 		/**
-		 * Whether or not to use the `.mjs` extension for JavaScript files. This makes it possible to preload JavaScript files in Safari, too (else only works in Chromium-based browsers).
-		 * Check if your provider has the correct MIME type for `.mjs` files before turning this on.
+		 * What preload strategy to use for JavaScript files to avoid waterfalls:
+		 * - `modulepreload` (default) - uses `<link rel="modulepreload">` to preload JavaScript files. Works only in Chromium-based browsers currently, and soon in Safari, too.
+		 * - `preload-js` - uses `<link rel="preload">` to preload JavaScript files. Works in all browsers but Firefox, and causes double-parsing of the script for Chromium-based browser.
+		 * - `preload-mjs` - uses `<link rel="preload">` to preload JavaScript files, but uses the `.mjs` extension. Works in Chromium-based browsers and Safari. Check of your provider/CDN has the correct MIME type for `.mjs` files before turning this on.
```

Comment:
```suggestion
		 * - `preload-mjs` - uses `<link rel="preload">` to preload JavaScript files, but uses the `.mjs` extension. Works in Chromium-based browsers and Safari. Check if your server or CDN sets the correct `Content-Type` header for `.mjs` files (text/javascript) before turning this on; browsers will error on a missing or incorrect `Content-Type` header.
```

Thank you, this overall comment is much clearer.